Gemfile

```diff
@@ -3,7 +3,9 @@
 source 'https://rubygems.org'
 ruby '>= 2.3.0', '< 2.5.0'
 
-gem 'pkg-config'
```

Comment:
Sure.

So, it turns out that `pkg-config` isn't smart enough to properly find the right libraries when used with Nanobox. Since it is only included for building `nokogiri`, and it's possible to manually provide the correct paths while installing that, leaving it out of Nanobox deploys is preferable (especially since the manual config seems to be flat ignored if `pkg-config` is present).

This change lets me exclude `pkg-config` from the install list. My understanding is that, unless explicitly excluded, all package groups are treated as dependencies, so this shouldn't affect other platforms at all. That's part of the reason for using a unique name for the group - to prevent it from affecting any users except those who exclude it intentionally.

I am more than happy to use another approach instead, if anyone is aware of one.